packages/toolkit/connector-kit/src/types.ts

```diff
@@ -209,6 +209,21 @@ export type EmailConnector = BaseConnector<ConnectorType.Email> & {
   sendMessage: SendMessageFunction;
 };
 
+const urlRegEx =
+  /(https?:\/\/)?(?:www\.)?[\w#%+.:=@~-]{1,256}\.[\d()A-Za-z]{1,6}\b[\w#%&()+./:=?@~-]*/;
+
+export const emailServiceBrandingGuard = z
+  .object({
+    fromName: z.string(),
```

Comment:
Guard for `fromName` is the same as `companyAddress`

Reply:
Good call.

Reply:
Updated.